API/Login.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+	ini_set('display_errors', 1);
+    ini_set('error_reporting', E_ALL);
+
+	$inData = getRequestInfo();
+
+	$servername = "localhost";
+	$username = "root";
+	$password = "";
+	$dbname = "myDB";
```

Comment:
Before this is merged, these should be updated to the credentials for the server, but maybe we can store them in an environment variable so that anyone looking at the source code is not able to see them. Accessing environment variables in php [should be straightforward](https://www.php.net/manual/en/reserved.variables.environment.php), and we can set the environment variables in the .bashrc on the server.

Reply:
This comment also applies to all the other instances of setting these fields.